release-notes/aws/v9.2.2.md

```diff
@@ -0,0 +1,37 @@
+## :zap: Giant Swarm Release 9.2.2 for AWS is now active for you! :zap:
+
+**If you are upgrading from 9.2.1, upgrading to this release merely updates the apps. It will *not* roll your nodes.**
+
+**It is not recommended to upgrade from 9.0.0 or older platform release, please only upgrade to 9.2.2 from 9.0.1 or newer once they are available.**
+
+Rationale for this is that 9.0.0 and older platform releases include apps which are still using Kubernetes 1.16 incompatible APIs
+
+This release includes multiple improvements to the NGINX Ingress Controller app.
+
+This version also includes an upgrade to the latest upstream NGINX IC 0.30.0, as well as improvements to other components (cluster-operator, kube-state-metrics, chart-operator) as detailed in the changelog.
```

Comment:
Aside from main value of this release, what other important changes are her? They are not on cluster-operator and chart-operator, right?

Reply:
I am missing the updates to cluster operator and chart operator in this changelog, were they actually changed?

also in this version IC version was not changed at all, there was just config changes, there was also not multiple changes there. I think I need a call for this.

Reply:
no, that was WIP release notes, unfinished copy pasted stuff from previous release notes, not ready for thorough review, I shared PR just to have others work on it while I was busy with other tasks (cleanup of broken apps without downtime risk + dvag support), but I continued today, and updated the notes, PTAL

Reply:
Resolved w Stevo's change.